src/bootstrap/dist.rs

```diff
@@ -894,8 +894,16 @@ impl Step for PlainSourceTarball {
 
         // Create the version file
         builder.create(&plain_dst_src.join("version"), &builder.rust_version());
-        if let Some(sha) = builder.rust_sha() {
-            builder.create(&plain_dst_src.join("git-commit-hash"), &sha);
+        // If there is information for one of them, then there must be
+        // information for the rest, too.
```

Comment:
won't block on this, but would be nice to avoid the unwraps here
```suggestion
        // If there is information for one of them, then there must be
        // information for the rest, too.
        // TODO: put this in a single `Option` so we don't have to use `unwrap()`
```

Reply:
Perhaps instead of making the `rust_sha_short` and `rust_commit_date` methods in [lib.rs#L1336-L1344](https://github.com/dawnofmidnight/rust/blob/tarball-commit-info/src/bootstrap/lib.rs#L1336-L1344), we could just expose the inner `Option<&Info>` from [channel.rs#L90-L95](https://github.com/dawnofmidnight/rust/blob/tarball-commit-info/src/bootstrap/channel.rs#L90-L95)?

Reply:
:+1: making `Info` public seems fine to me